source/as-stream.ts

```diff
@@ -37,7 +42,14 @@ export default function asStream(options: MergedOptions) {
 			return;
 		}
 
-		isFinished = true;
+		{
+			const read = proxy._read.bind(proxy);
+			proxy._read = (...args) => {
+				isFinished = true;
+
+				return read(...args);
+			};
+		}
```

Comment:
It's a pity there is no `read` event...

Reply:
After the first chunk is sent further `.pipe(...)` calls should throw.

Currently it throws if the `response` event was emitted.